ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -157,16 +160,34 @@ public class BottomSheetController: UIViewController {
         overflowView.backgroundColor = Colors.NavigationBar.background
         view.addSubview(overflowView)
 
+        view.addLayoutGuide(maxSheetHeightLayoutGuide)
```

Comment:
What happens if someone changes collapsedConentHeight? Is there badness to accumulating layout guides?

Reply:
But it's a var so this code path could get hit more than once. So then is there badness to calling addLayoutGuide multiple times with maxSheetHeightLayoutGuide. Or anything else in here where there is badness to potentially it getting repeated if collapsedContentHeight changes?

Reply:
This is in loadView. Are we looking at the same thing?

Reply:
Oh I see what happened. It's bad GitHub PR UI. When you expand the file you'll see this is in loadView.

Reply:
Oops my bad. Sorry for the confusion